__tests__/app.test.js

```diff
@@ -214,3 +204,75 @@ describe('POST - /api/reviews/:review_id/comments', () => {
+    test('PATCH /api/reviews/2 - status 200 - response review_id must be 2', ()=> {
+        return request(app).patch('/api/reviews/2').expect(200).send({inc_votes:-100}).then(result => {
+            expect(result.body.updatedReview[0].review_id).toBe(2)
+        })
+    })
+    test('/api/reviews/2 - status 200 - responds with an object containing 9 properties', ()=> {
+        return request(app).patch('/api/reviews/2').expect(200).send({inc_votes: -100}).then(result => {
+            expect(Object.keys(result.body.updatedReview[0]).length).toBe(9)
+        })
+    })
+    test('/api/reviews/2 - status 200 - responds with an object containing a votes property with an updated value of -95', ()=> {
+        return request(app).patch('/api/reviews/2').expect(200).send({inc_votes: -100}).then(result => {
+            expect(result.body.updatedReview[0].votes).toBe(-95)
+        })
+    })
+    test('/api/reviews/2 - status 200 - responds with an object containing a votes property with an updated value of 25', ()=> {
+        return request(app).patch('/api/reviews/2').expect(200).send({inc_votes: 20}).then(result => {
+            expect(result.body.updatedReview[0].votes).toBe(25)
+        })
+    })
+    test('/api/reviews/2 - status 200 - responds with an object containing 9 properties (review_id, title, review_body, designer, review_img_url, votes, category, owner and created_at)', ()=> {
+        return request(app).patch('/api/reviews/2').expect(200).send({inc_votes:-100}).then(result => {
+                expect(typeof result.body.updatedReview[0].review_id).toBe('number')
+                expect(typeof result.body.updatedReview[0].title).toBe('string')
+                expect(typeof result.body.updatedReview[0].review_body).toBe('string')
+                expect(typeof result.body.updatedReview[0].designer).toBe('string')
+                expect(typeof result.body.updatedReview[0].review_img_url).toBe('string')
+                expect(typeof result.body.updatedReview[0].votes).toBe('number')
+                expect(typeof result.body.updatedReview[0].category).toBe('string')
+                expect(typeof result.body.updatedReview[0].owner).toBe('string')
+                expect(typeof result.body.updatedReview[0].created_at).toBe('string')
+        })
+    })
+})
+
+describe('PATCH /api/reviews/nonsense', () => {
+    test('PATCH /api/reviews/nonsense - status 400 - with a message of "Bad Request', () => {
+        return request(app).patch('/api/reviews/nonsense').expect(400).send({inc_votes: -100}).then(result => {
+            expect(result.body.msg).toBe('Bad Request!')
+        })
+    })
+})
+describe('PATCH /api/reviews/100000', () => {
+    test('PATCH -/api/reviews/100000 - status 404 - with a "Review Not Found! message', () => {
+        return request(app).patch('/api/reviews/1000000').send({inc_votes: null}).expect(404).then(result => {
+            expect(result.body.msg).toBe('Review Not Found!')
+        })
+     })
+})
+
```

Comment:
We also want to add  a Status 400 test where the patch has an , incorrect body, e.g. inc_votes property is not a number